### PR TITLE
Geometry Enhancement 

### LINKED ISSUE #9: Populate Geometry Column of Osm_History Table
In order to extract accurate stats and use index , it is necessary to have geom column populated . Upon discussion for historical raw data extraction we are going to rely on insight ! Idea is to write a script to perform geometry calculation for way features and update correct geometry for node features ! Currently we are not focusing on relation type geometry calculation. Theirs and data outside loaded countries element geometry may remain null ! 
It will be one time update for all historical elements and May be we need to add that calculation part when we update our history table so that new rows will have geom column populated. 
cc: @omranlm , @ramyaragupathy

### PR DESCRIPTION
**Pull request by itskshitiz321**
- Creates Separate file for connection that can be used for other .py scripts 

- Has Database class for database operation with all the error handling , can be inherited to other .py script to avoid code duplication ( can be extended to different .py script if required )
- Has Geom Update Script 

- Resolve #9

### CODE REVIEW

File: geom_update.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                oeh.lat)
+                when oeh.type = 'way' then public.construct_geometry(oeh.nds,
+                oeh.id,
+                oeh."timestamp")
+                else 
+                    Null
+            end
+        where
+            oeh.action != 'delete'
+            and oeh.type != 'relation'"""
```
Comment (omranlm) [inline geom_update.py:121]: in this case, whenever we run this geom_update script we will update all osm_element_history table again, while we need to construct the ways geom and fix the nodes geom

we can add this condition to the node checking ..  oeh.geom = ST_MakePoint(oeh.lat, oeh.lon)

File: geom_update.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        
+        for i in range(int(start),int(end)):
+            query =base_query+ f""" and oeh.id={i}"""
```
Comment (omranlm) [inline geom_update.py:124]: update by batches per dates frequency 
 and oeh.timestamp between **start_date** and **end_date**

**Review by omranlm — CHANGES_REQUESTED** [review]
comments inline

File: geom_update.py (hunk 2 above)
Comment (itskshitiz321) [reply geom_update.py:124]: I don't want to repeat loop to go for same date . between will include <= and >= , is this a good query expression ?

**Review by omranlm — APPROVED** [review]
We just changed the frequency to monthly, and we can change it to a parameter for the script at one point as enhancement

**Pull request merged**